zzupy/model/eas.py

```
@@ -0,0 +1,758 @@
+            time_val = data.get(key)
+            if time_val:
+                time_str = str(time_val).strip().zfill(4)
+                try:
+                    schedule_time = PlainDateTime.parse_strptime(
+                        time_str, format="%H%M"
+                    ).time()
+                    data[key] = schedule_date.at(schedule_time).assume_tz(
+                        "Asia/Shanghai"
+                    )
+                except Exception:
+                    pass
```

Ruff (0.15.2)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)